Test/PcapDeviceTest.cs

```diff
@@ -35,12 +35,22 @@ namespace Test
     [NonParallelizable]
     public class PcapDeviceTest
     {
+        /// <summary>
+        /// Test that we can convert each TimestampType
+        /// </summary>
+        [Category("Timestamp")]
```

Comment:
If you want, I can create a custom attribute that automatically disable/enable unit tests based on Libpcap version

Reply:
See #233